91-appendixA.Rmd

```diff
@@ -205,7 +205,7 @@ Say you have a normal distribution with mean $\mu = 6$ and standard deviation $\
 
 **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What proportion of the area under the normal curve is less than 3? Greater than 12? Between 0 and 12?
 
-**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What is the 2.5th percentile of the area under the normal curve? The 95th percentile? The 100th percentile?
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What is the 2.5th percentile of the area under the normal curve? The 97.5th percentile? The 100th percentile?
```

Comment:
We should probably add in the solutions to these questions as well to the Learning Check Solutions Appendix.

Reply:
Haha. I didn't read far enough into the PR :).